app/constants/url.js

```diff
@@ -1 +1,12 @@
 export const GITHUB_URL = 'https://github.com/Real-Dev-Squad/website-my';
+export const RDS_MAIN_URL = 'http://realdevsquad.com/';
+export const RDS_WELCOME_URL = 'https://welcome.realdevsquad.com/';
+export const RDS_EVENTS_URL = 'http://realdevsquad.com/events.html';
+export const RDS_MEMBERS_URL = 'https://members.realdevsquad.com/';
+export const RDS_CRYPTO_URL = 'https://crypto.realdevsquad.com/';
+export const RDS_STATUS_URL = 'https://status.realdevsquad.com/';
+export const LOGOUT_URL = 'https://api.realdevsquad.com/auth/signout';
```

Comment:
Can we please remove this URL from here as we will be use `BASE_API_URL` with template strings for this purpose?

Reply:
This is rectified now!